idd/Energy+.idd.in

```diff
@@ -49467,13 +49467,106 @@ Coil:Cooling:Water:DetailedGeometry,
        \type object-list
        \object-list WaterStorageTankNames
-  N17; \field Design Water Temperature Difference
+  N17, \field Design Water Temperature Difference
        \type real
        \units deltaC
        \minimum> 0.0
        \note This input field is optional. If specified, it is used for sizing the Design Water Flow Rate.
        \note If blank or omitted, the Loop Design Temperature Difference value specified in Sizing:Plant
        \note object is used for sizing the Design Water Flow Rate.
+  N18; \field Design Water Inlet Temperature
+       \type real
+       \units C
+       \minimum> 0.0
+       \note This input field is optional. If specified, it is used for sizing the coil Design Geomtery
+       \note Parameters. If blank or omitted, the Design Loop Exit Temperature value specified in 
+       \note Sizing:Plant object is used for sizing the coil Design Geomtery Parameters.
+
+CoilSystem:Cooling:Water,
+       \memo Virtual container component that consists of a water cooling coil
+       \memo and its associated controls. This control object supports the
+       \memo available water coil types and may be placed directly on an
+       \memo air loop branch or in an outdoor air equipment list.
+       \min-fields 6
+   A1, \field Name
+       \required-field
+       \reference WaterCoilSystemName
+       \type alpha
+       \reference-class-name validBranchEquipmentTypes
+       \reference validBranchEquipmentNames
+       \reference-class-name validOASysEquipmentTypes
+       \reference validOASysEquipmentNames
+   A2, \field Air Inlet Node Name
+       \required-field
+       \type node
+   A3, \field Air Outlet Node Name
+       \required-field
+       \type node
+   A4, \field Availability Schedule Name
+       \note Availability schedule name for this system. Schedule value > 0
+       \note means the system is available.
+       \note If this field is blank, the system is always available.
+       \type object-list
+       \object-list ScheduleNames
+   A5, \field Cooling Coil Object Type
+       \type choice
+       \required-field
+       \key Coil:Cooling:Water
+       \key Coil:Cooling:Water:DetailedGeometry
+       \key CoilSystem:Cooling:Water:HeatExchangerAssisted
+   A6, \field Cooling Coil Name
+       \required-field
+       \type object-list
+       \object-list CoolingCoilsWater
+   A7, \field Dehumidification Control Type
+       \type choice
+       \key None
+       \key Multimode
+       \key CoolReheat
+       \default None
+       \note None = meet sensible load only
+       \note Multimode = activate water coil and meet sensible load.       
+       \note If no sensible load exists, and Run on Latent Load = Yes, and a latent
+       \note load exists, the coil will operate to meet the latent load.
+       \note If the latent load cannot be met the heat exchanger will be activated.
+       \note IF Run on Latent Load = No, the heat exchanger will always be active.
```

Comment:
Hmm, I don't think this is what we want. If control type = multimode and run on sensible = yes but run on latent = no, I would still expect the humidity control to switch on/off the HX as needed. The only thing run on latent = no indicates is that the coil shouldn't operate if there's only a latent load.